src/flutter/shell/platform/linux_embedded/surface/environment_egl.h

```diff
@@ -6,6 +6,9 @@
 #define FLUTTER_SHELL_PLATFORM_LINUX_EMBEDDED_SURFACE_ENVIRONMENT_EGL_H_
 
 #include <EGL/egl.h>
+#include <EGL/eglext.h>
```

Comment:
You shouldn't write a specific code for EGLStream in this file.

Reply:
I moved a specific code for EGLstream to `environment_egl_drm_eglstream.h`.